src/UI/walletConnect/WalletConnectLoginContainer/index.tsx

```diff
@@ -127,10 +173,47 @@ export const WalletConnectLoginContainer = ({
             ) : (
               <p className={generatedClasses.leadText}>{lead}</p>
             )}
+            {isWalletConnectV2 && activePairings && activePairings?.length > 0 && (
+              <div className={generatedClasses.pairsContainer}>
```

Comment:
I think this JSX can be moved into a separate component and mounted here, to abstract away some code